api/TextTrackList.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextTrackList",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "23"
```

Comment:
This matches api.HTMLMediaElement.textTracks, but `TextTrack` says 18 which I doubt is correct. Could you re-validate that data and if it's wrong update it in this PR as well?

Reply:
Yep, collector's saying TextTrack is supported since Chrome 23.  Updates coming up!